en/02_Developer_Guides/15_Customising_the_Admin_Interface/08_ReactJS_Redux_and_GraphQL.md

```diff
@@ -149,7 +151,17 @@ documentation available all over the web. We recommend:
 * [Getting Started with Redux](https://egghead.io/courses/getting-started-with-redux)
 * [The React Apollo docs](http://dev.apollodata.com/react/)
 
-# The Injector API
+## Build tools and using Silverstripe React components {#using-cms-react-components}
+
+Silverstripe CMS includes react, redux, graphql, apollo, and many other thirdparty dependencies already, which are exposed using [webpack's expose-loader plugin](https://webpack.js.org/loaders/expose-loader/) for you to use as [webpack externals](https://webpack.js.org/configuration/externals/).
+
+There are also a lot of React components and other custom functionality (such as the injector, mentioned below) available for reuse. These are exposed in the same way.
+
+The recommended way to access these dependencies is by using the [@silverstripe/webpack-config npm package](https://www.npmjs.com/package/@silverstripe/webpack-config). The documentation in the readme for that package explains how to use it.
+
+If you are not using webpack to transpile your javascript, see if your build tooling has an equivalent to webpack's `externals` configuration. Alternatively, instead of `import`ing these depdencies, you can access them on the `window` object (for example the injector module is exposed as `window.Injector`).
```

Comment:
```suggestion
If you are not using webpack to transpile your javascript, see if your build tooling has an equivalent to webpack's `externals` configuration. Alternatively, instead of `import`ing these dependencies, you can access them on the `window` object (for example the injector module is exposed as `window.Injector`).
```

Reply:
Done